tooling/src/hypothesistooling/__main__.py

```diff
@@ -457,7 +457,7 @@ def standard_tox_task(name):
 standard_tox_task("nose")
 standard_tox_task("pytest43")
 
-for n in [20, 21, 22, 111]:
+for n in [20, 21, 22, 30, 111]:
```

Comment:
We can drop the `20` here as well.

In fact looking at the download page suggests that 2.1 is either already out of support or about to be, so I'd remove that from our test matrix too - we're not going to break it in the next few weeks.

Reply:
IMHO, anything below 2.2 should be removed

Reply:
The DSF recommends that for third-party *apps*, but as a library I'm happy to support 1.11 until it reaches EOL in April.